javaslang/src/main/java/javaslang/collection/Stream.java

```diff
@@ -1250,6 +1250,10 @@ default int length() {
         return Collections.scanRight(this, zero, operation, Iterator::toStream);
     }
 
+    @Override default Stream<T> shuffle() {
```

Comment:
We do a newline after annotations (in 99.9% of the cases). But please leave it as-is. I will reformat the whole codebase soon according to our formatter. Maybe I will take that opportunity to customize the Sputnik/Checkstyle rules then.